ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/behaviorTree/BehaviorTreeNodeDefinition.java

```diff
@@ -51,8 +52,12 @@ public BehaviorTreeNodeDefinition(CRDTInfo crdtInfo, WorkspaceResourceDirectory
 
       this.saveFileDirectory = saveFileDirectory;
 
+      childrenModification = new LatestTimestampModifiable(crdtInfo);
```

Comment:
The data modifiable is extended, but the children are kept as a separate modifiable in order to handle topology synchronization separately from data, to improve understandability in the ROS2BehaviorTreeSubscription class.